docs/content/quickstart/installation.md

```diff
@@ -87,72 +87,72 @@ k3s:
 
 Save the config file as `config.yaml` as we will use it later in the process.
 
-Note:
-- The `stages.initramfs` block will configure the `kairos` user (default) with the `kairos` password. Note, the `kairos` user is already configured with sudo.
+**Note**:
+- The `stages.initramfs` block will configure the Kairos user (default) with the Kairos password. Note, the Kairos user is already configured with `sudo`.
 - `authorized_keys` can be used to add additional keys to the user in order to SSH into
 - `hostname` sets the machine hostname
 - `dns` sets the DNS for the machine
 - `k3s.enabled=true` enables `k3s`. 
 
 {{% notice note %}}
 
-Several configuration can be added at this stage. [See the configuration reference](/reference/configuration) for further reading.
+Several configurations can be added at this stage. [See the configuration reference](/reference/configuration) for further reading.
 
 {{% /notice %}}
 
 ### Deployment
 
 {{% notice note %}}
 
-Below there are instruction using the `kairos-cli`.  To install by logging over SSH into the box see [Manual install](/installation/manual) or [Interactive install](/installation/interactive) for driving the installation manually from the console.
+Below there are instruction using the Kairos CLI.  To install by logging over SSH into the box, see the [Manual installation](/installation/manual) section or the [Interactive installation](/installation/interactive) section for driving the installation manually from the console.
 
 {{% /notice %}}
 
-To trigger the installation process via QR code you need to use the `kairos-cli`, the cli is currently available only for Linux and Windows. It can be downloaded from the releases artifact:
+To trigger the installation process via QR code, you need to use the Kairos CLI. The CLI is currently available only for Linux and Windows. It can be downloaded from the release artifact:
```

Comment:
```suggestion
To trigger the installation process via QR code, you need to use the Kairos CLI. The CLI is currently only available for Linux and Windows. It can be downloaded from the release artifact:
```

Reply:
> Suggested change 
> To trigger the installation process via QR code, you need to use the Kairos CLI. The CLI is currently available only for Linux and Windows. It can be downloaded from the release artifact:
> To trigger the installation process via QR code, you need to use the Kairos CLI. The CLI is currently only available for Linux and Windows. It can be downloaded from the release artifact:


Let's keep this version: The CLI is currently available only for Linux and Windows.